lib/tasks/deploy.rake

```diff
@@ -25,6 +25,16 @@ task :deploy_to, [:destination] do |t, args|
   )
 end
 
+# Runs multiple up migrations.
+#
+# This task is mostly run in Heroku during the deploy. No need to run this locally.
+#
+# Usage: rake migrate_up[20150226124214, 20150226124215, 20150226124216]
```

Comment:
Why do we need an array? Passing just one migrates up to that version (or down to that).